wasm/packages/okapi/package.json

```diff
@@ -1,46 +1,77 @@
 {
   "name": "@trinsic/okapi",
-  "version": "1.0.0-preview.27",
```

Comment:
Can we revert this to simply 1.0.0? We're always overriding this when publishing, no need to specify actual production version here

Reply:
I'm fine with `1.0.0`, will do. I just figured the `preview.27` was vastly out of date.

Reply:
Yeah that was from the time we checked in the actual version and had no automation